tests/fixtures/gremlin-console/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM tinkerpop/gremlin-console:latest
```

Comment:
Just curious - why did you choose gremlin-console over gremlin-sever for the test fixture?

Reply:
Ah, that's probably a bit of misleading organization. The test fixture is actually the peer directory with the tinkerpop server. The gremlin-console Dockerfile and supporting files aren't actually test fixtures.  Per the text in the Contributing.md file, they're there to give people a way to manually connect to the graph database and run ad hoc queries to investigate system behavior and debug during development, the way you'd use a browser manually connected to Neo4J to debug system interactions with a Neo4J database.  I welcome suggestions on a better place to locate it. It might be better to place it as a root subdirectory to the project rather than in test fixtures, given that it's not really a fixture.  Good catch, and any preferences?

Reply:
Makes sense - no need to change it.